### PR TITLE
Switch 1.17 to dedicated Rails 5.1.4 repository

### PR DESCRIPTION
**Pull request by ehelms**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.17
* [ ] 1.16
* [ ] 1.15

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

File: repoclosure/yum_el7.conf [hunk 1]
```diff
@@ -48,9 +48,9 @@ baseurl=http://yum.puppetlabs.com/el/7/PC1/$basearch/
 name=el7-puppet-5
 baseurl=https://yum.puppetlabs.com/puppet5/el/7/$basearch/
 
-[el7-tfm-ror51]
-name=el7-tfm-ror51
-baseurl=https://copr-be.cloud.fedoraproject.org/results/@theforeman/tfm-ror51/epel-7-$basearch/
+[el7-tfm-ror51-5.1.4]
```
Comment (ekohl) [inline repoclosure/yum_el7.conf]: Would it make sense to keep this on `el7-tfm-ror51` to keep the repoclosure changes minimal?

File: foreman-release-scl/foreman-release-scl.spec [hunk 2]
```diff
@@ -1,6 +1,6 @@
 Name:     foreman-release-scl
 Version:  4
-Release:  1%{?dist}
+Release:  2%{?dist}
```
Comment (ekohl) [inline foreman-release-scl/foreman-release-scl.spec]: Needs a changelog

Comment (ekohl) [reply foreman-release-scl/foreman-release-scl.spec]: Disregard that, this is 1.17 which still uses tags.

**Comment by ehelms** [comment]
Updated based on both comments

File: foreman-release-scl/foreman-release-scl.spec [hunk 3]
```diff
@@ -29,6 +29,9 @@ install -m 644 %{SOURCE0} %{buildroot}%{_sysconfdir}/yum.repos.d/
 %config %{_sysconfdir}/yum.repos.d/*.repo
 
 %changelog
+* Mon Apr 09 2018 Eric D. Helms <ericdhelms@gmail.com> 4-2
```
Comment (ekohl) [inline foreman-release-scl/foreman-release-scl.spec]: Looks like you didn't see my update - I think this is not needed for the 1.17 branch.

**Comment by ekohl** [comment]
> Oh yea, good call, updated

The release bump was still needed though :)

**Comment by ehelms** [comment]
It's the final update...

**Comment by ehelms** [comment]
Closing based on IRC discussions with @ekohl about how to handle this for 1.18+

**Pull request closed without merging**